frontend/views/containers/modals/CreateGroup.vue

```diff
@@ -128,7 +128,7 @@ export default {
           groupName: this.form.groupName,
           groupPicture: this.form.groupPicture,
           sharedValues: this.form.sharedValues,
-          incomeProvided: +this.form.incomeProvided, // ensure this is a number
+          mincomeAmount: +this.form.mincomeAmount, // ensure this is a number
```

Comment:
Re the line below, `incomeCurrency` also needs to be changed project-wide to `mincomeCurrency` to match the settings validation for the new action you created

Reply:
Done